src/components/refactor/dialogs/load-modification/load-modification-dialog.js

```diff
@@ -120,7 +120,11 @@ const LoadModificationDialog = ({
     }, [reset, emptyFormData]);
 
     return (
-        <FormProvider validationSchema={schema} {...methods}>
+        <FormProvider
+            validationSchema={schema}
+            {...methods}
+            removeOptionnel={true}
```

Comment:
`optional` not `optionnel` which is french
And by the way do we want to pass our props through FormProvider ? 
This is strange

Reply:
Ok `validationSchema` is also our prop, maybe move its declaration befor `{...methods}` order is important here.

Reply:
done